03-objects-and-composition/exercise/03. Car Factory

```diff
@@ -0,0 +1,29 @@
+function carFactory(car) {
+    let newCar = {};
+    newCar.model = car.model;
+    newCar.engine = {power: 0, volume: 0};
+ 
+    if (car.power <= 90) {
+        newCar.engine.power = 90;
+        newCar.engine.volume = 1800;
+    } else if (car.power > 90 && car.power <= 120) {
```

Comment:
The else if can be (car.power <= 120), because if it's below or equal to 90 it will enter the first case. The last case can be only an else.

Reply:
Great observation! Thank you!